embedded-graphics/src/primitives/line/thick_points.rs

```diff
@@ -122,11 +134,12 @@ impl ParallelsIterator {
             left_error: 0,
             right: Bresenham::new(start_point),
             right_error: 0,
-            next_side: Side::Right,
+            next_side,
+            stroke_offset,
         };
 
         // Skip center line on left side iterator
-        self_.next_parallel(Side::Left);
+        self_.next_parallel(skip_side);
```

Comment:
Is `skip_side` required? The `match` expression above assigns `skip_side` to be `next_side.swap()`.